### PR TITLE
Add missing aria labels and roles

### LINKED ISSUE #84: Missing aria labels and roles
Buttons and other elements are missing the `aria` labels and roles.
See: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

### PR DESCRIPTION
**Pull request by uuykay**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting)
- [ ] Refactoring (no functional changes)
- [ ] Documentation changes
- [x] Other - Please describe:

## Fixes
#84 

Issue Number:

## Other information
Improvements to accessibility with role and aria-labels to buttons as requested in issue #84. I added 3 new tests too!

### CODE REVIEW

**Comment by btzr-io** [comment]
@uuykay Awesome, thanks for the contribution :V:

Can look at the input elements as well? if you have the time ( not required to merge )

File: src/components/toolbar/button.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ class Button extends Component {
     } = this.props
     const button = (
       <button
+        role="button"
+        aria-label={this.props.ariaLabel || title}
```
Comment (btzr-io) [inline src/components/toolbar/button.js]: simplify: `aria-label={ariaLabel || title}`

**Comment by btzr-io** [comment]
Also could you update change log with your changes?

File: src/components/toolbar/button.js (hunk 1 above)
Comment (uuykay) [reply src/components/toolbar/button.js]: No problem, I was thinking of adding ariaLabel in the object destructuring, but thinking from a performance optimisation point of view, I feel like it adds a run-time step which isn't necessary. However I do accept that it is more readable so I will make the change to align with your codebase.

**Comment by uuykay** [comment]
> @uuykay Awesome, thanks for the contribution ✌️
> 
> Can look at the input elements as well? if you have the time ( not required to merge )

No problem, I can take a look at this tonight when I'm off from work.

**Comment by uuykay** [comment]
> Also could you update change log with your changes?

Sure

**Comment by btzr-io** [comment]
can you fix the conflicts ?

**Comment by uuykay** [comment]
> can you fix the conflicts ?

Done. Although they were really simple conflicts - like a space, it could have easily been done by yourself.

**Comment by btzr-io** [comment]
@uuykay Lol, Sorry I didn't see the file just the `github` warning, let me know if you are still working on this or if I should merge now, thanks :v:

**Comment by uuykay** [comment]
All good, I'll do the input adjustments, but sorry I couldn't work on it last night. I'll spend a little bit of time today to do it after work and send a message.

**Comment by btzr-io** [comment]
@uuykay Don't worry, take your time :+1:

**Comment by uuykay** [comment]
@btzr-io Aria labels added to inputs along with roles / contenteditable attributes. I've updated the changelog too.

File: src/components/render.js [hunk 2]
```diff
@@ -366,7 +366,7 @@ class CanvasRender extends Component {
     return (
       <React.Fragment>
         <Toolbar
-         localize={localize}
+          localize={localize}
```
Comment (btzr-io) [inline src/components/render.js]: We don't use this anymore

File: src/components/toolbar/index.js [hunk 3]
```diff
@@ -97,13 +97,19 @@ class Toolbar extends Component {
...
 
   render() {
     // Component Props
-    const { allowFullScreen, showControls, toggleFullscreen, renderError, localize } = this.props
+    const {
+      allowFullScreen,
+      showControls,
+      toggleFullscreen,
+      renderError,
+      localize,
```
Comment (btzr-io) [inline src/components/toolbar/index.js]: Remove this line

File: src/components/toolbar/navigation.js [hunk 4]
```diff
@@ -91,6 +91,9 @@ class Navigation extends Component {
           step={1}
           max={totalPages}
           type="number"
+          aria-label="Go to page number"
```
Comment (btzr-io) [inline src/components/toolbar/navigation.js]: We should use `Localize[string...]` if posisble

File: src/components/toolbar/zoom.js [hunk 5]
```diff
@@ -89,6 +88,9 @@ class ZoomControls extends Component {
         <input
           type="text"
           title="Zoom"
+          aria-label="Zoom to percentage value"
```
Comment (btzr-io) [inline src/components/toolbar/zoom.js]: we should use `Localize[string...]` if possible

**Pull request merged**